server/server.js

```diff
@@ -51,64 +57,95 @@ module.exports = function(__dirname) {
 	app.use(renderer.handle.bind(renderer))
 	app.use(api)
 
+	/**
+	  * Annotations
+	**/ 
+
+	var requireAdmin = (fn) =>
```

Comment:
I prefer to use arrow functions for curried functions, mostly because it's more clear in my view